playwright/utils/getSitemapLocations.js

```diff
@@ -43,11 +44,23 @@ async function getSitemapLocations(baseUrl) {
 
 // VA.gov sitemaps have a lot of urls in them. Helper function for things that
 // may want to parallelize checking that list (broken links, a11y, etc.)
-function splitPagesIntoSegments(pages, count) {
-  const segmentSize = Math.ceil(pages.length / count)
-  return new Array(count).fill().map((_, index) => {
-    return pages.slice(index * segmentSize, (index + 1) * segmentSize)
+function splitPagesIntoBatches(pages, batchCount) {
```

Comment:
These two functions could possible be dried up; they're somewhat similar. The first is a rework of the existing batching of pages and is used within a single instance. The second function `getPagesSlice` is used to provide a given review instance its set of files. The output of the second function ultimately gets passed to the first to be further broken up into batches.

Reply:
not a blocker for this PR because we don't actually run the full a11y scan right now, but the initial stub (`playwright/tests/a11y.spec.js`) used the old `splitPagesIntoSegments` for its test batching. that will need to be reworked to match this.